kimchi/src/constants.rs

```diff
@@ -0,0 +1,12 @@
+/// Number of constraints produced by the permutation argument.
```

Comment:
a module documentation would be nice (`//!`)

Reply:
Added module doc in https://github.com/o1-labs/proof-systems/pull/974/commits/b2bb15aa2556a627152b1b800e7ebc5cb239a682